arsclib-utils/src/main/kotlin/app/revanced/arsc/logging/Logger.kt

```diff
@@ -0,0 +1,7 @@
+package app.revanced.arsc.logging
```

Comment:
Maybe relying on a logging facade is better than having these interfaces in patcher and arsclib-utils.